dynomitemanager-core/src/main/java/com/netflix/dynomitemanager/storage/RedisStorageProxy.java

```diff
@@ -128,15 +127,24 @@ private void startPeerSync(String peer, int port) {
 
         localRedisConnect();
```

Comment:
How do you know this connect succeeded

Reply:
We get a warning underneath but effectively we do not know and it does not matter. Even if this is null, then the start peer syncing will fail, so the warm-up will fail.